### DISCUSSION TITLE
The Faucets want ETH on the mainnet. I don't have it.

### DISCUSSION THREAD

**Discussion by md-exitcode0** [Cyfrin/foundry-full-course-cu#2834] [General]
I tried every one available in the course and even more but they all require the ETH on the mainnet. I could do the metamask smart contract deployment on the 12th lesson in the second course. Can't do the course. I understand that this is optional but I want to do everything.

**Comment by EngrPips** [comment]
Try this https://faucets.chain.link/sepolia

**Reply by md-exitcode0** [reply]
Doesn't work, wants link or something when I want sepolia eth. Any other recommendations.

**Reply by EngrPips** [reply]
Please show me what you are getting it use to work fine for me.

**Reply by md-exitcode0** [reply]
The one you gave me wants 1 $LINK on the Mainnet so I used another faucet and got the sepolia eth, so for everyone here is the guranteed working link: https://cloud.google.com/application/web3/u/3/faucet/ethereum/sepolia

**Reply by EngrPips** [reply]
Thanks for the heads-up

**Reply by Nenepo** [reply]
> The one you gave me wants 1 $LINK on the Mainnet so I used another faucet and got the sepolia eth, so for everyone here is the guranteed working link: https://cloud.google.com/application/web3/u/3/faucet/ethereum/sepolia

I tried this but it still needs ETH in my mainnet

**Reply by LordAdhaar** [reply]
It allows you get one time in 24hours. Try after that fixed time

**Comment by Mramy9** [comment]
Hello trust all is well please I keep getting the same issue after trying to send 0.05 ETH to my metamask wallet on zksync first transaction after clicking on receive 0.05 Sepolia ETH this message appears "To help prevent automated abuse, please verify that your wallet address has at least 0.001 ETH on Ethereum Mainnet (see FAQ). If you believe your request was flagged in error, please contact us." i waited for 20 minutes as instructed in the video but I keep getting the same issue for the past 4 hours keep waiting after 20 minutes or more than that but keep encountering the same issue please kindly I need your help on this thank you.

**Reply by EngrPips** [reply]
You should bridge `sepolia` ether to `zksyn-sepolia` ether if you are having hard time get `zkysnc-sepolia` ether from faucet. You can bridge ethereum sepolia eth to zksync-sepolia eth using this bridge https://portal.zksync.io/bridge/

**Reply by Mramy9** [reply]
Ok thank you.

**Reply by EngrPips** [reply]
Anytime, Friend.

**Reply by Mramy9** [reply]
> Anytime, Friend.

Please I'm still having trouble sending testnet faucet from both https://faucets.chain.link/sepolia and https://cloud.google.com/application/web3/u/3/faucet/ethereum/sepolia and others faucet link please assist.

**Reply by EngrPips** [reply]
Use this faucet https://sepolia-faucet.pk910.de/
